src/index.html

```diff
@@ -11,7 +11,146 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form method="post" action="https://mate-academy-form-lesson.herokuapp.com/create-application">
+
+      <fieldset class="fieldset-group">
+        <legend>Personal information</legend>
+        <label class="input-group" for="surname">Surname:
```

Comment:
```suggestion
        <label class="input-group">
          Surname:
```
`for` and `id` are redundant